src/bci_build/package/golang/README.md.j2

```diff
@@ -58,26 +86,69 @@ To run the application tests inside a container, use the following command:
 $ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} go test -v
 ```
 
-**Note:** The Golang image should be used as a build environment. For runtime, self-contained Go binaries should use a `scratch` image and for applications that require external dependencies use the `bci-base` image.
+### Building from SLE BCI
 
-## Additional tools
+The Go image should be used as a build environment. That means
+the compiler does not need to be shipped as part of the images that are
+distributed. Instead, it is recommended that the Go image is used as the
+builder image only.
 
-The following additional tools are included in the image:
+The [SLE BCI General Purpose Base Containers](https://opensource.suse.com/bci-docs/documentation/general-purpose-bci/)
+images offer four different options here, depending on your exact requirements.
 
-- go{{ image.version }}-race
-- make
-- git-core
+```Dockerfile
+# Build the application using the {{ image.pretty_name }} Container Image
+FROM {{ image.pretty_reference }} as build
 
-{% if image.version.endswith('-openssl') %}
-## FIPS 140
+WORKDIR /app
 
-The image includes a FIPS 140 enabled Go toolchain using OpenSSL.
+# pre-copy/cache go.mod for pre-downloading dependencies and only
+# redownloading them in subsequent builds if they change
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
 
-To restrict all TLS configuration to FIPS-approved settings, add the following line:
+COPY . ./
+
+# Make sure to build the application with CGO disabled.
+# This will force Go to use some Go implementations of code
+# rather than those normally supplied by the host operating system.
+# You need this for scratch images as those supporting libraries
+# are not available.
```

Comment:
```suggestion

```

Reply:
I don't understand the deletion request, the sentence is half finished then?